taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css

```diff
@@ -84,11 +87,17 @@ Styleguide Objects.Section
 /* Modifers: Style */
 
 /* Modifers: Style: Dark & Light */
+/* FP-1470: Add cross-style code and variables when third style is added */
 
-.o-section--style-dark,
-.o-section--style-dark::before {
+.o-section--style-dark {
   color: var(--global-color-primary--xx-light);
   background-color: var(--global-color-primary--xx-dark);
+  box-shadow: 50vw 0 var(--global-color-primary--xx-dark),
+              -50vw 0 var(--global-color-primary--xx-dark);
+}
+.o-section--style-dark + .o-section--style-dark {
+  border-top: var(--border-width) solid
+              var(--global-color-primary--xx-light);
 }
```

Comment:
**Explanation.**
For __dark__ sections:
- Extend existing background using a `box-shadow`, not a `::before` (which had more styles later in file).
- Move the `border-top` up from below, and use variables.

Reply:
Outdated since recent commit. Resolving comment.